<p>Redis Session Should set Expiration Time.</p>
<h2>Noncompliant Code Example</h2>
<pre>
    public static void saveSession(String id, Map session) {
        Jedis jedis = null; // Noncompliant {{Jedis should set the expiration time}}
        try {
            jedis = RedisUtil.getJedis();
            jedis.set(id.getBytes(), serialize(session));
        } catch (Exception e) {
            logger.error("保存session错误：", e);
            e.printStackTrace();
        } finally {
            jedis.close();
            logger.error(session.toString());
        }
    }
</pre>
<h2>Compliant Solution</h2>
<pre>
    public static void saveSession(String id, Map session) {
        Jedis jedis = null;
        try {
            jedis =RedisUtil.getJedis();
            jedis.set(id.getBytes(), serialize(session));
            jedis.expire(id.getBytes(), RedisUtil.SESSION_TIME_OUT);  // Compliant
        } catch (Exception e) {
            logger.error("保存session错误：",e);
            e.printStackTrace();

        } finally {
            jedis.close();
            logger.error(session.toString());
        }
    }
</pre>
